### DISCUSSION TITLE
request for Full Width

### DISCUSSION THREAD

**Discussion by konser80** [rxhanson/Rectangle#1338] [Ideas]
Hello! I'm using vertical display (but now it doesn't matter)

1. my request is for a shortcut - Full Width, and don't change height
2. is it possible (in settings) to add an option for... set the left point to -1px, right point to +1px. So I will not see 1px white line around the app.

Thank you

**Comment by rxhanson** [comment]
Sorry for the delayed response.

> 1. my request is for a shortcut - Full Width, and don't change height

This isn't something I plan on adding into Rectangle at this time, but would be fine merging a pull request for it as long as it is not in the UI. As an alternative, you could create a custom shortcut in [Rectangle Pro](https://rectangleapp.com/pro) (paid) that does this:

<img width="931" alt="image" src="https://github.com/rxhanson/Rectangle/assets/13651296/1e0d9162-ad0a-4453-997f-80fdb54bfcf2">

> 2. is it possible (in settings) to add an option for... set the left point to -1px, right point to +1px. So I will not see 1px white line around the app.

Unfortunately, even if this were allowed in the app, macOS would place the window to the size that fits the screen without going over.

**Reply by MacDada** [reply]
> As an alternative, you could create a custom shortcut in [Rectangle Pro](https://rectangleapp.com/pro)

I've done that and it works, thanks!

Although I also think that there could be a proper option in the UI, instead of searching for a workaround:
`Maximize Width` (analogous to `Maximize Height`).
